src/LibraryManagement.Application/CommandHandlers/Authentication/LoginCommandHandler.cs

```diff
@@ -0,0 +1,75 @@
+﻿using LibraryManagement.Application.Commands.Authentication;
+using LibraryManagement.Domain.Entities.Authentication;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace LibraryManagement.Application.CommandHandlers.Authentication;
+
+public class LoginCommandHandler : IRequestHandler<LoginCommand, string>
+{
+    private readonly UserManager<User> _userManager;
+    private readonly IConfiguration _configuration;
+    private User? _user;
+
+    public LoginCommandHandler(UserManager<User> userManager, IConfiguration configuration)
+    {
+        _userManager = userManager;
+        _configuration = configuration;
+    }
+
+    public async Task<string> Handle(LoginCommand command, CancellationToken cancellationToken)
+    {
+        _user = await _userManager.FindByEmailAsync(command.Email);
+
+        if (_user != null && await _userManager.CheckPasswordAsync(_user, command.Password))
```

Comment:
It would be best if you separated the second condition into a variable

Reply:
Updated